topsort-api-v2.yml

```diff
@@ -148,6 +150,34 @@ paths:
           $ref: '#/components/responses/BadRequest'
         402:
           $ref: '#/components/responses/PaymentRequired'
+  /events:
+    post:
+      tags:
+        - Events
+      summary: Report events
+      operationId: reportEvents
+      requestBody:
+        description: >
+          Use the `/events` endpoint to notify Topsort about significant consumer interactions on the marketplace app:
+          impressions — ad promotions become visible to the consumer;
+          clicks — the consumer clicks on an ad promotion;
+          and purchases — the consumer buys some products.
```

Comment:
Very fancy em-dash

Reply:
I thought in some markdown apps double dash gets automatically replaced with an en-dash or em-dash, but not in Redocly.